src/pages/dapps.tsx

```diff
@@ -126,252 +116,154 @@ import wallet from "@/public/images/wallet.png" // width=300
 
 const Page = (props: ChildOnlyProp & FlexProps) => (
   <Flex
-    as={MainArticle}
-    direction="column"
-    align="center"
-    mx="auto"
-    w="full"
-    {...props}
-  />
+    asChild
+    className={cn("mx-auto w-full flex-col items-center", props.className)}
+  >
+    <MainArticle {...props} />
+  </Flex>
 )
 
 const Content = (props: ChildOnlyProp) => (
-  <Box py={4} px={8} w="full" {...props} />
+  <div className="w-full px-8 py-4" {...props} />
 )
 
 const OptionContainer = (props: ChildOnlyProp) => (
   <Flex
-    direction={{ base: "column", lg: "row" }}
-    justify="center"
-    px={8}
-    mb={8}
-    w={{ base: "full", lg: "auto" }}
+    className="mb-8 w-full flex-col justify-center px-8 lg:w-auto lg:flex-row"
     {...props}
   />
 )
 
 const Option = (
   props: Pick<ButtonProps, "children" | "onClick"> & { isActive: boolean }
 ) => {
-  const tableBoxShadow = useToken("colors", "tableBoxShadow")
-
   return (
     <Button
       variant="outline"
-      display="flex"
-      alignItems="center"
-      justifyContent={{ base: "center", lg: "flex-start" }}
-      boxShadow={props.isActive ? tableBoxShadow : `none`}
-      color={props.isActive ? "primary.base" : "text"}
-      borderColor={props.isActive ? "primary.base" : "text"}
-      borderRadius="2rem"
-      height="auto"
-      w={{ base: "full", lg: "auto" }}
-      my={2}
-      mx={{ base: 0, lg: 2 }}
-      py={4}
-      px={6}
-      transition="none"
-      _hover={{
-        color: "primary.base",
-        borderColor: "primary.base",
-      }}
-      _active={{ bg: "transparent" }}
+      className={`my-2 flex w-full items-center justify-center rounded-4xl border px-6 py-4 transition-none lg:mx-2 lg:w-auto lg:justify-start ${
+        props.isActive
+          ? "border-primary text-primary shadow-table-box"
+          : "border-body text-body"
+      } hover:border-primary hover:text-primary active:bg-transparent`}
       {...props}
     />
   )
 }
 
 const OptionText = (props: ChildOnlyProp) => (
-  <Text
-    as="span"
-    fontSize={{ base: "md", md: "2xl" }}
-    textAlign="center"
-    fontWeight={{ base: "semibold", md: "normal" }}
-    lineHeight="100%"
+  <span
+    className="text-center text-md font-semibold md:text-2xl md:font-normal"
     {...props}
   />
 )
 
 const ButtonPrimary = (props: Pick<ButtonProps, "children" | "onClick">) => (
-  <Button py={2} px={3} borderRadius="0.25em" {...props} />
+  <Button className="rounded px-3 py-2" {...props} />
```

Comment:
in order to be consistent with our design system, lets use the default values
```suggestion
  <Button {...props} />
```